staging/nvidia/charts/grafana/dashboards/dcgm-exporter-dashboard.json

```diff
@@ -100,7 +100,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "DCGM_FI_DEV_GPU_TEMP{instance=~\"${instance}\", gpu=~\"${gpu}\"}",
+          "expr": "DCGM_FI_DEV_DEC_UTIL{instance=~\"${instance}\", gpu=~\"${gpu}\"}",
```

Comment:
That looks wrong - we are replacing the GPU device temperature with decoder utilization here.

Reply:
oops sorry, I replaced the wrong one, should be replacing `DCGM_FI_DEV_GPU_UTIL` but perhaps it's still not the right metric..